### PR TITLE
fix(docs): format code sample in README - @gregfenton

### PR DESCRIPTION
**Pull request by gregfenton**
Some markup cleanup and minor hints to the instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -275,26 +275,26 @@ describe("Test firestore", () => {
...
+   ```
+   
+1. Update your application code to connect to the emulators (where your code calls `firebase.initializeApp(...)`), , updating the localhost ports as appropriate from the `emulators` values in the previous step:
 
    ```js
    const shouldUseEmulator = window.location.hostname === "localhost"; // or other logic to determine when to use
    // Emulate RTDB
    if (shouldUseEmulator) {
-     fbConfig.databaseURL = `http://localhost:9000?ns=${fbConfig.projectId}`;
+     fbConfig.databaseURL = `http://localhost:_9000_?ns=${fbConfig.projectId}`;
```
Comment (prescottprue) [inline README.md]: Is there a reason for the underscores here? Seems like it would make things not work

File: README.md [hunk 2]
```diff
@@ -310,7 +310,7 @@ describe("Test firestore", () => {
 
    // Emulate Firestore
    if (shouldUseEmulator) {
-     firestoreSettings.host = "localhost:8080";
+     firestoreSettings.host = "localhost:_8080_";
```
Comment (prescottprue) [inline README.md]: Same here

File: README.md (hunk 1 above)
Comment (gregfenton) [reply README.md]: Was meant to show italics, indicating that this is a modifiable value.  But I hadn't considered the copy/paste issue. . .

File: README.md (hunk 2 above)
Comment (gregfenton) [reply README.md]: Ditto.

Comment (gregfenton) [reply README.md]: (BTW: when I installed the firebase emulator, I had to go with non-default ports.  So I went with :6000, :6001, :6002, ...)

File: README.md (hunk 1 above)
Comment (prescottprue) [reply README.md]: If we remove the underscores I can merge

**Pull request merged**